server/lobbyconnection.py

```diff
@@ -771,7 +772,8 @@ def command_game_host(self, message):
 
         assert isinstance(self.player, Player)
 
-        title = cgi.escape(message.get('title', ''))
+        default_title = "%s' game".format(self.player.login)
+        title = html.escape(message.get('title', True) or default_title)
```

Comment:
Shouldn't this be
```suggestion
        title = html.escape(message.get('title', False) or default_title)
```
or even

```suggestion
        title = html.escape(message.get('title') or default_title)
```